app/src/main/res/values-fa/strings.xml

```diff
@@ -0,0 +1,173 @@
+<resources>
+    <string name="app_name">یوزرلند</string>
```

Comment:
Hi @ahangarha,  I'm another developer on the UserLAnd team here.  Thanks for the PR, we appreciate your efforts.  I just wanted to add my comments on the code changes.  What do you think about keeping `"UserLAnd"` untranslated?  (This includes all other lines have `"UserLAnd"` in them, such as line 155)

My reasoning is that regardless of what language their phone is set to, see the app name as `UserLAnd` in the Google Play Store.  Keeping the text in the app to show `"UserLAnd"` will keep the naming consistent in both places.

Reply:
Hi. I totally agree that name shouldn't be translated. But I haven't translated UseLAnd but I just transliterate it. It is very much common as far as I see. For example Telegram messaging app has Persian title when the phone default language is Persian.

But if you want to keep the text as is, for example because you write it in a very particular way, that is a decision to be made on your side. I will follow what your policy is.

Reply:
Transliteration is probably fine. All we have for checking your work is Google Translate haha, which made us slightly skeptical of the translation as it was coming out as "User Name" or "Usherland".